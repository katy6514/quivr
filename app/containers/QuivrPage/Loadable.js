/**
 * Asynchronously loads the component for QuivrPage
 */

import loadable from 'utils/loadable';

export default loadable(() => import('./index'));
