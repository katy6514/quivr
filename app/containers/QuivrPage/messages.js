/*
 * QuivrPage Messages
 *
 * This contains all the text for the QuivrPage container.
 */
import { defineMessages } from 'react-intl';

export const scope = 'app.containers.QuivrPage';

export default defineMessages({
  header: {
    id: `${scope}.header`,
    defaultMessage: "Here's what's in your quivR",
  },
});
