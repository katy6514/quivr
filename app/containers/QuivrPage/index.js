/*
 * QuivrPage
 *
 * This is the first thing users see of our App, at the '/' route
 *
 */

import React from 'react';
import { FormattedMessage } from 'react-intl';
import messages from './messages';

function QuivrPage() {
  return (
    <h1>
      <FormattedMessage {...messages.header} />
    </h1>
  );
}

export default QuivrPage;
