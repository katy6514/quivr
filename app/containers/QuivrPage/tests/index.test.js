import React from 'react';
import { render } from 'react-testing-library';
import { IntlProvider } from 'react-intl';

import QuivrPage from '../index';

describe('<QuivrPage />', () => {
  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <IntlProvider locale="en">
        <QuivrPage />
      </IntlProvider>,
    );
    expect(firstChild).toMatchSnapshot();
  });
});
