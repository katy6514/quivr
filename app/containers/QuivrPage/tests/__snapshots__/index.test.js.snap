// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<QuivrPage /> should render and match the snapshot 1`] = `
<h1>
  <span>
    This is the QuivrPage container!
  </span>
</h1>
`;
